core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -967,6 +967,137 @@ paths:
                  $ref: '#/components/schemas/ErrorReport'
         500:
            $ref: '#/components/responses/RawlsInternalError'
+  /api/admin/bucketMigration/workspaces/{workspaceNamespace}/{workspaceName}:
```

Comment:
I think I would prefer to keep it under "workspaces". So  `/api/admin/workspaces/{workspaceNamespace}/{workspaceName}/bucketMigration`.

Reply:
Okay! And then an equivalent `api/admin/billing/{projectId}/bucketMigration` for the billing project based APIs?

Reply:
Oh, I hadn't gotten any further in the PR, ha! Now that you call out the billing use case, perhaps since the APIs can work either via workspace or billing project it is cleaner to leave it the way you have it with `/api/admin/bucketMigration/`.